contrib/colab/issue-report.ipynb

```diff
@@ -0,0 +1,361 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "PyTorch_XLA_Debug",
+      "provenance": [],
+      "collapsed_sections": [],
+      "machine_shape": "hm"
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "accelerator": "TPU"
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "-5-xIyOwIk45",
+        "colab_type": "text"
+      },
+      "source": [
+        "Install PyTorch Nightly packages and set up the backend version."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "sPJVqAKyml5W",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "VERSION = \"nightly\"  #@param [\"20200220\",\"nightly\", \"xrt==1.15.0\"]\n",
```

Comment:
Can we get rid of `xrt==1.15.0` and follow our newer colabs?

Reply:
Yep!